cypress/page/payment-step.page.ts

```diff
@@ -0,0 +1,22 @@
+class PaymentPage {
+
+    private payByBankWireButton: string;
+    private confirmButton: string;
+    public confirmeMessage:string;
```

Comment:
you are breaking the encapsulation principle here, please tho a method to get the confirmationMessage

Reply:
Done!